client/components/credit-card/docs/example.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CreditCard from '..';
+import Button from 'components/button';
+
+class CreditCards extends Component {
+	state = {
+		cards: [
+			{
+				lastDigits: '4242',
+				cardType: 'visa',
+				name: 'James Smith',
+				expiry: '2023-05-31',
+			},
+			{
+				lastDigits: '4444',
+				cardType: 'mastercard',
+				name: 'Jean Davis',
+				expiry: '2021-01-31',
+			},
+		],
+	};
+
+	render() {
+		return (
+			<Fragment>
+				{ this.state.cards.map( ( card, i ) => (
+					<CreditCard key={ `${ card.lastDigits }_${ i }` } card={ card } selected={ 0 === i } />
+				) ) }
+				<CreditCard>
+					<Button compact style={ { margin: 20 } }>
+						{ this.props.translate( 'Add another credit card' ) }
+					</Button>
+				</CreditCard>
+			</Fragment>
+		);
+	}
+}
+
+const LocalizedCreditCards = localize( CreditCards );
+LocalizedCreditCards.displayName = 'CreditCard';
```

Comment:
Since you are using ES6 class already, you can just put `displayName` as a static data property right in the class definition as
```
static displayName = 'CreditCard';
```
A example: https://github.com/Automattic/wp-calypso/blob/master/client/components/forms/form-password-input/index.jsx#L19

Also, I assume it should be `CreditCards` instead of the singular form?

With that, line 50 ~ 53 can then be simplified as just `export default localize( CreditCards )`.

Reply:
Thanks for testing!

This `displayName` is intended to appear as the title of the component in devdocs. Though the example contains multiple `<CreditCard>`s, the component is still singular, following the pattern of other examples: https://github.com/Automattic/wp-calypso/blob/34365c471f635ea24e90e7a3a976b52df1f9e3f3/client/components/button/docs/example.jsx#L20-L21

And since `localize` transforms the `displayName`, I set it on the localized component rather than the original. Open to other ways of doing this, but I hadn't come across any.